vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1402,7 +1409,8 @@ public void getCurrentUser(@NotNull VimeoCallback<User> callback) {
      * @param callback the callback to be invoked when the request finishes
      */
     public void getCurrentUser(@Nullable String filter, @NotNull VimeoCallback<User> callback) {
-        getContent(Vimeo.ENDPOINT_ME, CacheControl.FORCE_NETWORK, GetRequestCaller.USER, null, null, filter, callback);
+        getContent(Vimeo.ENDPOINT_ME, CacheControl.FORCE_NETWORK, GetRequestCaller.USER, null,
```

Comment:
A few things here:
1. Please format this file
2. Our current preferred style is to put each parameter on a new line if the method call will exceed the allowed line length
3. Please add @SuppressWarnings("WeakerAccess") to this method to silence the lint warning.